ext/zip/tests/oo_readonly_properties.phpt

```diff
@@ -0,0 +1,71 @@
+--TEST--
+Test that ZipArchive properties are read-only
+--EXTENSIONS--
+zip
+--FILE--
+<?php
+
+$zip = new ZipArchive();
+
+try {
+    $zip->lastId = 1;
```

Comment:
Can you please check that stuff like `+= 1` is also prohibited? I think you also need a get_property_ptr handler...

Reply:
Ah, I see that there already is one, nevermind.

Reply:
I added a a test for `+= 1`, just in case